### PR TITLE
[DDW-442] Fix failing apply-node-update test

### PR DESCRIPTION
**Pull request by daniloprates**
This PR fixes failing `apply-node-update` test.

---

### Basics

- [ ] PR is updated to the most recent version of target branch (and there are no conflicts)
- [ ] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`yarn run test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn run dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn run package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn run flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn run lint`)
- [ ] Text changes are proofread and approved (Jane Wild)
- [ ] There are no missing translations (running `yarn run manage:translations` produces no changes)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn run storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly documented and commented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-rendering
- [ ] Any code that only works in Electron is neatly separated from components

### Testing
- [ ] New feature / change is covered by acceptance tests
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature / change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@daniloprates please merge latest `develop` into your branch - you will see that @DominikGuzei added a couple of very cool test-related helpers which you could use as well!

File: features/step_definitions/node-update-notification-steps.js [hunk 1]
```diff
@@ -71,3 +72,9 @@ Then(/^I should see the Daedalus window close$/, async function () {
     expect(windowCount).to.equal(0);
   }, 1500);
 });
+
+Then(/^I should see Daedalus process not running$/, { timeout: 30000 }, async function () {
```
Comment (DominikGuzei) [inline features/step_definitions/node-update-notification-steps.js]: I wonder how this should fix anything if it is not used by any feature file? 😄

**Comment by DominikGuzei** [comment]
For me all tests are passing now! (But sometimes i need to restart the backend + `yarn install` to make it work)

**Comment by DominikGuzei** [comment]
@MarcusHurney please review this PR!

**Comment by MarcusHurney** [comment]
The same test that was previously failing on my environment refactor PR and recent versions of `develop` is still failing for this PR. The actual apply node update test is passing, but all proceeding tests fall apart in a cascading manner once this `paper wallets certificate success generation` test runs the step `I enter paper wallet recovery phrase` and fails/freezes.

screen shot 2018-11-15 at 9 18 47 pm
screen shot 2018-11-15 at 9 19 05 pm

**Comment by DominikGuzei** [comment]
@MarcusHurney please jump in and debug this as i cannot reproduce the issue on my machine!

**Comment by nikolaglumac** [comment]
I'm seeing the same issue as @MarcusHurney...

**Comment by nikolaglumac** [comment]
@DominikGuzei can you try deleting your `node_modules` and run the tests again? Just to see if that will make a difference...

**Comment by DominikGuzei** [comment]
@nikolaglumac i just did a clean install and successfully ran the test suite on this branch:
screenshot 2018-11-19 um 11 55 39

**Comment by nikolaglumac** [comment]
@DominikGuzei let's have a call about this in 5 mins!

**Comment by nikolaglumac** [comment]
@DominikGuzei on my machine all tests are passing now - and I really like the improvements you did 🎉 
screen shot 2018-11-24 at 18 51 05

Do you think this PR is now ready to be merged? 🙏

**Pull request merged**